### PR TITLE
PLAT-6614 handle kms.enabled

### PR DESCRIPTION
**Pull request by miguelhar**
[PLAT-6614](https://dominodatalab.atlassian.net/browse/PLAT-6614)

Ignore kms changes to avoid resources destruction.

### CODE REVIEW

**Review by fraenkel — APPROVED** [review]

File: examples/kms/main.tf [hunk 1]
```diff
@@ -19,6 +19,8 @@ module "domino_eks" {
     enabled = true
   }
   kms = {
-    enabled = true
+    enabled = false
```
Comment (Secretions) [inline examples/kms/main.tf:22]: Couldn't we just lifecycle ignore this so we can have the default we want? Seems really unfortunate to debut with a default other than our preference due to pre-release usage.

**Review by Secretions — CHANGES_REQUESTED** [review]

File: examples/kms/main.tf (hunk 1 above)
Comment (miguelhar) [reply examples/kms/main.tf:22]: Given that this is a very intrusive setting I rather we are explicit when enabling it. But I do agree having the lifecycle ignore its a good additional settings given that we are not looking to change the behavior once set

Comment (Secretions) [reply examples/kms/main.tf:22]: Ok, if we legitimately want the default to be off that's fine. I thought we were sacrificing a prior decision about defaults for compatibility.

**Review by Secretions — APPROVED** [review]
Long as we proactively want kms to be default, I'm fine with this.

**Review by ddl-dclegg — APPROVED** [review]
I'm good with this route forward.

**Review by Secretions — APPROVED** [review]

**Pull request merged**